lib/blockscore/base.rb

```diff
@@ -15,11 +17,11 @@ def initialize(options = {}, &block)
     def attributes
       return @attributes if @loaded
       force!
-      @attributes
+      attributes
     end
 
     def id
-      @attributes.fetch(:id, nil)
+      attributes.fetch(:id, nil)
```

Comment:
Yes. If you want a default to `nil` (or something not dynamically computed already present in the instances `Hash#default`) you want `Hash#[]`.